package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bulla-network/contracts",
-  "version": "0.0.11-bulla-finance-beta-0.2",
+  "version": "0.0.11-frendlend-beta-0.2",
```

Comment:
hmm could have maybe incremented the last number. We don't know which is more recent.